docs/_generate/index.html

```diff
@@ -727,32 +727,82 @@ <h3 class='space-bottom1'><a href='#types-function' title='link to function'>Fun
 
-          <div class='col12 pad1x space-bottom2'>
-            <ul>
-              <li class='pad0y'>
-                <div><span class='code'>stops</span><em class='pad1x quiet'>Required <a href='#types-array'>array</a>.</em></div>
-                <div class='pad1x'>Functions are defined in terms of input and output values. A set of one input value and one output value is known as a "stop."</div>
-              </li>
-              <li class='pad0y'>
-                <div><span class='code'>property</span><em class='pad1x quiet'>Optional <a href='#types-string'>string</a>.</em></div>
-                <div class='pad1x'>If specified, the function will take the specified feature property as an input. See <a href="#types-function-zoom-property">Zoom Functions and Property Functions</a> for more information.</div>
-              </li>
-              <li class='pad0y'>
-                <div><span class='code'>base</span><em class='pad1x quiet'>Optional <a href='#types-number'>number</a>. Default is <%= ref.function.base.default %>.</em></div>
-                <div class='pad1x'>The exponential base of the interpolation curve. It controls the rate at which the function output increases. Higher values make the output increase more towards the high end of the range. With values close to 1 the output increases linearly.</div>
-              </li>
-              <li class='pad0y'>
-                <div><span class='code'>type</span><em class='pad1x quiet'>Optional <a href='#types-string'>enum</a>. One of <var>identity</var>, <var>exponential</var>, <var>interval</var>, <var>categorical</var>.</em></div>
-                <div class='pad1x'>
-                    <div class=''></span><var>identity</var> functions return their input as their output.</div>
-                    <div class=''><var>exponential</var> functions generate an output by interpolating between stops just less than and just greater than the function input. This is the default for <span class='icon smooth-ramp quiet micro space-right indivne' title='continuous'></span> properties.</div>
-                    <div class=''></span><var>interval</var> functions return the output value of the stop just less than the function input. This is the default for <span class='icon step-ramp quiet micro space-right indivne' title='discrete'></span> properties.</div>
-                    <div class=''></span><var>categorical</var> functions return the output value of the stop equal to the function input.</div>
-                </ul>
-              </li>
-            </ul>
+          <div class='col12 pad1x'>
+            <div class="col12 clearfix pad0y pad2x space-bottom2">
+              <div><span class='code'><a id="function-stops" href="#function-stops">stops</a></span></div>
+              <div><em class='quiet'>Required <a href='#types-array'>array</a>.</em></div>
```

Comment:
stops are not required if `type === 'identity'`, right? We remove the _Required_ label I think and explain that caveat here.